go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/docker/go-units
```

Comment:
I'm thinking if this repository should also live in the `moby` org; wondering what's best; add a `go.mod` already, or wait with adding until after we moved; wdyt?

In either case, may be better to do it separate

Reply:
Right, let's move this to moby first.

The go.mod is added for selfish reasons only -- so I can actually run `go test`.

Reply:
👍 

reminds me, I also want to look at https://github.com/tonistiigi/units, and see where the gaps (and overlaps) are between the two, as both end up in our dependency tree; perhaps we can unify them

(I know there's some possibly "bad" choices in `docker/go-units`, so maybe there's reasons for one over the other, but 🤷‍♂️)

Reply:
Filed https://github.com/docker/go-units/issues/41 about move to under moby org. Once done we can move from there.

Looking into https://github.com/tonistiigi/units

Reply:
OK, Tonis' repo is about printing/output, and does not cover conversion/input.

Reply:
Yup; it's "the other side of go-units", so mostly complimentary; but in the same area, so it could make sense to have these together (not urgent, but an option).

I checked with Tonis, and he wasn't _against_, so it's an option we can consider (not "MUST", but _can_)

Reply:
> it could make sense to have these together 

Perhaps it's better to keep it separate, as they solve complementary, but different tasks.